### ISSUE TITLE
problematic target_entropy initialization in HIL_SERL

### ISSUE THREAD

**Issue by SitheralS** [huggingface/lerobot#1871]
in the document modeling_sac.py, the temperature loss is defined as such:

        temperature_loss = (-self.log_alpha.exp() * (log_probs + self.target_entropy)).mean()

However, the target_entropy is initialized as such when not explicitly configured:

            if self.target_entropy is None:
                dim = continuous_action_dim + (1 if self.config.num_discrete_actions is not None else 0)
                self.target_entropy = -np.prod(dim) / 2

I think the issue is that self.target_entropy should be initialized as self.target_entropy = np.prod(dim) / 2 rather than its opposite.
